src/state/board/state/index.types.ts

```diff
@@ -15,6 +15,7 @@ interface State<A extends SerializedAttachment, P extends SerializedPage> {
     redoStack?: StackAction[]
     strokeUpdates?: StrokeUpdate[]
     transformStrokes?: TransformStrokes
+    activeTextfield?: ActiveTextfield
```

Comment:
This field should rather be inside `drawing` state. There is no need for it to persist across sessions, is there?

Reply:
It is omitted during serialisation so it doesn't persist, and since its a stroke which is handled similarly to the transformStrokes i think it is in the right place

Reply:
transformStrokes should also be in the `drawing` state 😄